localstack/services/install.py

```diff
@@ -118,6 +118,13 @@
 )
 TERRAFORM_BIN = os.path.join(dirs.static_libs, f"terraform-{TERRAFORM_VERSION}", "terraform")
 
+# Java Test Jar Download (used for tests)
+TEST_LAMBDA_JAVA = os.path.join(config.dirs.var_libs, "localstack-utils-tests.jar")
```

Comment:
nit: We could also construct this path dynamically, when `install_lambda_java_testlibs()` gets called. If we keep only constants at the root scope (and avoid access to `config`), it might help us to introduce dynamic configs over time. (No need to change in this PR, though - we can pick this up in some future iteration.. 👍 ).